### PR TITLE
More multiple args related fixes

### PR DESCRIPTION
**Pull request by rafie**

### CODE REVIEW

File: RLTest/utils.py [hunk 1]
```diff
@@ -61,3 +62,54 @@ def Blue(data):
...
+
+    if isinstance(modules, list) and len(modules) > 1:
+        n = len(modules) - len(modulesArgs)
+        if n > 0:
+            modulesArgs.extend([['']] * n)
+    n = len(defaultArgs) - len(modulesArgs)
+    if n > 0:
+        modulesArgs.extend([['']] * n)
+
+    if defaultArgs and len(modulesArgs) != len(defaultArgs):
```
Comment (MeirShpilraien) [inline RLTest/utils.py:106]: why len(modulesArgs)  need to be the same as len(defaultArgs) ?

File: RLTest/utils.py [hunk 2]
```diff
@@ -61,3 +62,54 @@ def Blue(data):
...
+
+    fixed_modulesArgs = copy.deepcopy(defaultArgs)
+    if not modulesArgs:
+        return fixed_modulesArgs
+
+    if isinstance(modules, list) and len(modules) > 1:
+        n = len(modules) - len(modulesArgs)
+        if n > 0:
+            modulesArgs.extend([['']] * n)
+    n = len(defaultArgs) - len(modulesArgs)
```
Comment (MeirShpilraien) [inline RLTest/utils.py:102]: The set of module_args and defaultArgs might not be the same and might not join, so I do not understand this extension ..

File: RLTest/redis_std.py [hunk 3]
```diff
@@ -25,19 +25,8 @@ def __init__(self, redisBinaryPath, port=6379, modulePath=None, moduleArgs=None,
...
-            self.modulePath = None
-        if isinstance(moduleArgs, list):
-            self.moduleArgs = moduleArgs
-        else:
-            self.moduleArgs = [moduleArgs]
-            if isinstance(modulePath, list) and len(modulePath) > 1:
-                self.moduleArgs.extend([''] * (len(modulePath) - 1))
-
+        self.modulePath = fix_modules(modulePath)
+        self.moduleArgs = fix_modulesArgs(self.modulePath, moduleArgs)
```
Comment (MeirShpilraien) [inline RLTest/redis_std.py:29]: Why it need to be fixed again?

File: RLTest/__main__.py [hunk 4]
```diff
@@ -356,23 +356,17 @@ def __init__(self):
...
                     sys.exit(1)
-                module_args = []
-                for args in self.args.module_args:
-                    module_args += [' '.join(args) if args else '']
-            else:
-                module_args = ' '.join(self.args.module_args)
 
-        Defaults.module_args = module_args
+        Defaults.module = fix_modules(self.args.module)
+        Defaults.module_args = fix_modulesArgs(Defaults.module, self.args.module_args)
```
Comment (MeirShpilraien) [inline RLTest/__main__.py:369]: Fixed again?

File: RLTest/utils.py (hunk 1 above)
Comment (rafie) [reply RLTest/utils.py:106]: `modulesArgs` and `defaultArgs` are now of the form `[ ['arg',...], ['arg', ...], ...]` where each inner list is for its own module.

File: RLTest/utils.py (hunk 2 above)
Comment (rafie) [reply RLTest/utils.py:102]: This is needed if `moduleArgs` is longer than `defaultArgs`, in which case we're not padding the defaults and instead return an error.

File: RLTest/redis_std.py (hunk 3 above)
Comment (rafie) [reply RLTest/redis_std.py:29]: We get `modulePath` and `moduleArgs` from the constructor, thus we need to get them to the standard representation.

File: RLTest/__main__.py (hunk 4 above)
Comment (rafie) [reply RLTest/__main__.py:369]: `Defaults`, `Env`, and `StandardEnv` have independent module/args handling mechanisms (`Env` does consider `Defaults`, `StandardEnv` does not). So we need to fix those args in every constructor seperately.

**Review by MeirShpilraien — APPROVED** [review]

**Review by MeirShpilraien — APPROVED** [review]

**Pull request merged**